qemu/qemu.go

```diff
@@ -45,8 +45,8 @@ type Machine struct {
 
 // Device describes a device to be created by qemu.
 type Device struct {
```

Comment:
Most if not all devices today support vhost and multi queue capability. So at the device level we should specify if vhost is desired and how many queues are needed. Typically the caller can set the queues to number of vCPU's in guest.

The one challenge would be to figure out how the fd's created for all of the different devices map into the final QEMU invocation, where there are multiple devices with fd's.